python/ray/train/tests/test_huggingface_predictor.py

```diff
@@ -93,6 +94,37 @@ def test():
     ray.get(test.remote())
 
 
+def create_checkpoint():
+    with tempfile.TemporaryDirectory() as tmpdir:
+        model_config = AutoConfig.from_pretrained(model_checkpoint)
+        model = AutoModelForCausalLM.from_config(model_config)
+        tokenizer = AutoTokenizer.from_pretrained(tokenizer_checkpoint)
+        checkpoint = HuggingFaceCheckpoint.from_model(model, tokenizer, path=tmpdir)
+        return HuggingFaceCheckpoint.from_dict(checkpoint.to_dict())
```

Comment:
Ah no - because it's in a temporary directory (which will be deleted once we exit the context), we use `to_dict()` to serialize the checkpoint into memory. I can add a comment